x-pack/plugins/serverless_search/public/application/components/web_crawlers/elastic_managed_web_crawler_coming_soon.tsx

```diff
@@ -0,0 +1,293 @@
+
+import { BACK_LABEL } from '../../../../common/i18n_string';
+
+export const ElasticManagedWebCrawlersCommingSoon: React.FC = () => {
+  const {
+    application: { navigateToUrl },
+  } = useKibanaServices();
+
+  const assetBasePath = useAssetBasePath();
+  const webCrawlerIcon = assetBasePath + '/web_crawlers.svg';
+  const horizontalSteps: EuiStepsHorizontalProps['steps'] = [
+    {
+      title: '',
+      status: 'incomplete',
+      onClick: () => {},
+    },
+    {
+      title: '',
+      status: 'incomplete',
+      onClick: () => {},
+    },
+  ];
+  return (
+    <EuiFlexGroup alignItems="center" direction="column">
+      <EuiFlexItem>
+        <EuiPanel paddingSize="l" hasShadow={false} hasBorder>
+          <EuiFlexGroup
+            alignItems="center"
+            justifyContent="center"
+            direction="column"
+            gutterSize="l"
+          >
+            <EuiFlexItem>
+              <EuiButtonEmpty
+                data-test-subj="serverlessSearchElasticManagedWebCrawlerEmptyBackButton"
+                iconType="arrowLeft"
+                onClick={() => navigateToUrl(`./`)}
+              >
+                {BACK_LABEL}
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiIcon size="xxl" type={webCrawlerIcon} />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle>
+                <h2>
+                  {i18n.translate('xpack.serverlessSearch.elasticManagedWebCrawlerEmpty.title', {
+                    defaultMessage: 'Elastic managed web crawlers',
+                  })}
+                </h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiBadge color="accent">Coming soon</EuiBadge>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText textAlign="center" color="subdued">
+                <p>
+                  {i18n.translate(
+                    'xpack.serverlessSearch.elasticManagedWebCrawlerEmpty.description',
+                    {
+                      defaultMessage:
+                        "We're actively developing Elastic managed web crawlers, that won't require any self-managed infrastructure. You'll be able to handle all configuration in the UI. This will simplify syncing your data into a serverless Elasticsearch project. This new workflow will have two steps:",
+                    }
+                  )}
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexGroup
+              alignItems="stretch"
+              justifyContent="center"
+              direction="column"
+              gutterSize="s"
+            >
+              <EuiFlexItem>
+                <EuiPanel color="subdued">
+                  <EuiFlexItem grow={false}>
+                    <EuiStepsHorizontal
+                      css={css`
+                        pointer-events: none;
```

Comment:
I've been [discussing](https://elastic.slack.com/archives/C7QC1JV6F/p1730811554918959) this issue with the EUI team. And this a11y issue can be avoid using the [inert](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert) HTML attribute to prevent _click_ and _focus_ events for all this content. In addition using the [role="presentation"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) attribute we don't expose it to the accessibility tree.

With all this  `<EuiStepsHorizontal>` component is not interactive with the keyboard nor the mouse:

![CleanShot 2024-11-06 at 09 26 11](https://github.com/user-attachments/assets/9ca83465-38ec-48bf-97aa-a823590955e0)


```
<EuiStepsHorizontal
   css={css`
      pointer-events: none;
   `}
    steps={horizontalSteps}
    size="s"
    role="presentation"
    // @ts-ignore
    inert=""
/>
```